website/docs/reference/glossary.md

```diff
@@ -7,40 +7,40 @@ sidebar_label: Glossary
+| [Parent Stack](/core-concepts/components)                                                             | The Stack configuration which defines all components for an environment                                                                                                             |
+| [Real Component](/core-concepts/components)                                                           | A component that is instantiated in a Parent Stack                                                                                                                                  |
+| [Stack Manifest](/core-concepts/stacks)                                                               | Stack manifests are YAML files in which the configuration for all Atmos stacks and components are defined)                                                                          |
+| [Stacks](/core-concepts/stacks)                                                                       | Atmos stacks are configurations that function like blueprints to describe all [components](/core-concepts/components)\nAn Atmos stack can be defined in one or more stack manifests |
+| [Terraform "Child Modules"](https://developer.hashicorp.com/terraform/language/modules#child-modules) | Any terraform module that is called from another module. A child module does not have terraform state.                                                                              |
+| [Terraform "Root Module"](https://developer.hashicorp.com/terraform/language/modules#child-modules)   | These are components. It's any top-level terraform module with a state backend. This is where "terraform" is run.                                                                   |
+| [Vendoring](/core-concepts/components/vendoring)                                                      | A mechanism of making a copy of the 3rd party components in your repository.                                                                                                        |
+| [Namespace](/core-concepts/stacks)                                                                    | A prefix for all resources in a Stack                                                                                                                                               |
+| [Tenant](/core-concepts/stacks)                                                                       | A logical grouping of resources. In AWS we use the Tenant to represent the Organizational Unit (OU).                                                                                |
+| [Environment](/core-concepts/stacks)                                                                  | A location where resources are deployed (e.g. `us-east-1`). [See Disambiguation](#disambiguation)                                                                                   |
```

Comment:
This link doesn't seem to go anywhere

https://atmos.tools/reference/glossary#disambiguation